test/fs.test.js

```diff
@@ -161,4 +161,3 @@ describe('FileSystem based storage provider', function() {
     });
   });
 });
-
```

Comment:
That behavior happens once in a while from the editor. I had no control on it. Not sure why.
I will fix it in a different PR.